acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -19,17 +19,15 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.            f"Total time for the {model_type} model: {total_time}ms.",
</pre><p>We can cut some digits the printed total_time and averaged_time:</p><pre class="ql-syntax" spellcheck="false">f'{total_time:.2f}'
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>